tests/io/test_postgis.py

```diff
@@ -470,11 +519,68 @@ def test_read(self, example_trips, conn_postgis):
 
         try:
             trips.as_trips.to_postgis(table, conn_string)
-            trips_db = ti.io.read_trips_postgis(sql, conn_string)
+            trips_db = ti.io.read_trips_postgis(sql, conn_string, index_col="id")
             assert_frame_equal(trips, trips_db)
         finally:
             del_table(conn, table)
 
+    def test_non_standard_column_names(self, example_trips, conn_postgis):
+        """Test renaming handled by read_trips_gpd()."""
+        trips = example_trips.copy()
+        conn_string, conn = conn_postgis
+        table = "trips"
+        sql = f"SELECT * FROM {table}"
+        rename_dict = {
+            "started_at": "start_time",
+            "finished_at": "end_time",
+            "user_id": "USER",
+            "origin_staypoint_id": "ORIGIN",
+            "destination_staypoint_id": "DEST",
+        }
+        trips.rename(rename_dict, inplace=True)
+        try:
+            trips.as_trips.to_postgis(table, conn_string)
+            tpls_db = ti.io.read_trips_postgis(sql, conn_string, index_col="id", **rename_dict)
+            assert_frame_equal(example_trips, tpls_db)
+        finally:
+            del_table(conn, table)
+
+    def test_write_with_geometry(self, example_trips, conn_postgis):
+        """Test if write of trips with geometry creates correct schema in database."""
+        mp1 = MultiPoint([(0.0, 0.0), (1.0, 1.0)])
+        mp2 = MultiPoint([(2.0, 2.0), (3.0, 3.0)])
+        trips = gpd.GeoDataFrame(example_trips, copy=True, geometry=[mp1, mp2, mp2], crs="EPSG:4326")
+        conn_string, conn = conn_postgis
+        table = "positionfixes"
```

Comment:
I think the table name doesn't really matter but this should probably be "trips"